test/Telegram.Bot.Tests.Integ/Polls/PollMessageTests.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Telegram.Bot.Requests;
+using Telegram.Bot.Tests.Integ.Framework;
+using Telegram.Bot.Types;
+using Telegram.Bot.Types.Enums;
+using Xunit;
+
+namespace Telegram.Bot.Tests.Integ.Polls
+{
+    [Collection(Constants.TestCollections.NativePolls)]
+    [TestCaseOrderer(Constants.TestCaseOrderer, Constants.AssemblyName)]
+    public class PollMessageTests : IClassFixture<PollTestsFixture>
+    {
+        private ITelegramBotClient BotClient => _fixture.BotClient;
+
+        private readonly PollTestsFixture _classFixture;
+        private readonly TestsFixture _fixture;
+
+        public PollMessageTests(TestsFixture fixture, PollTestsFixture classFixture)
+        {
+            _fixture = fixture;
+            _classFixture = classFixture;
+        }
+
+        [OrderedFact(FactTitles.ShouldSendPoll)]
+        [Trait(Constants.MethodTraitName, Constants.TelegramBotApiMethods.SendPoll)]
+        public async Task Should_Send_Poll()
+        {
+            string pollQuestion = "Poll question";
+            string[] pollOptions =
+            {
+                "Poll option 1",
+                "Poll option 2",
+            };
+
+            SendPollRequest sendPollRequest = new SendPollRequest(
+                _fixture.SupergroupChat,
+                pollQuestion,
+                pollOptions
+            );
+
+            Message message = await BotClient.MakeRequestAsync(sendPollRequest);
```

Comment:
All of our tests use the methods defined on the interface.

```suggestion
            Message message = await BotClient.SendPollAsync(
                /* chatId: */ _fixture.SupergroupChat.Id,
                /* question: */ "Should we use a better question in here?",
                /* options: */ new[] {"YES", "Nah! It's fine."}
            );
```